zulipterminal/ui_tools/views.py

```diff
@@ -580,7 +580,7 @@ def __init__(self, controller: Any) -> None:
                     (max_widths[1], urwid.Text(", ".join(binding['keys'])))
                               ], dividechars=2),
                 tlcorner='', brcorner='', trcorner='', blcorner='',
-                rline=' ', lline=' ', bline='-', tline='',
```

Comment:
If we're removing this, I'm not sure what the point of the `LineBox` is now?

Reply:
As in, L577-585 are to be removed?

Reply:
I meant that the `LineBox` 'wrapper' around the `Columns` is rather less useful now; there is a left- and right- padding there, but I think that would be easier as a padding around the entire 'table' (rather than each entry), or just as spaces to the left and right of each entry.